point_cloud2_viewer:
  resolution: 0.05
  range:
    x: 10.0
    y: 10.0
  background_color:
    r: 200
    g: 200
    b: 200
  point_color:
    r: 0
    g: 0
    b: 0
  chart:
    enable: true
    range: 1.0
    angle: 45.0
    color:
      r: 0
      g: 0
      b: 255
